### ISSUE TITLE
Fine-tuning from pretrain

### ISSUE THREAD

**Issue by grendelaglaeca** [alinvdu/reproduce-dream-diffusion#1]
Hi, I'm trying to fine-tune the model using the DreamDiffusion eeg_lmd pipeline and your eeg pretrain checkpoint but the model doesn't seem to be training, i. e. the vlb loss is oscillating without any signs of converging even after 300 epochs. I was wondering if you tried it too and if you managed to reproduce it successfully?

**Comment by alinvdu** [comment]
Yes, from what I remember the total loss and clip loss were going down rapidly. Not sure specifically about vlb_loss.
